tests/introspection/Mac/MacApiProtocolTest.cs

```diff
@@ -31,6 +31,7 @@ protected override bool Skip (Type type, string protocolName)
 			switch (protocolName) {
 			case "NSSecureCoding":
 				switch (type.Name) {
+				case "CNContactFormatter": // Not declared in header file
```

Comment:
really ? or an incorrect comment ?

Conformance is documented -> https://developer.apple.com/documentation/contacts/cncontactformatter

and we are we claiming it in our bindings ?

`interface CNContactFormatter : NSSecureCoding {`

Normally that comment is used when `CNContactFormatter` says it conforms **at runtime** but the headers don't match - but in such case it's not in the bindings.

Please double-check :)

Reply:
I checked the header files:

/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Contacts.framework/Versions/A/Headers/CNContactFormatter.h
```
NS_CLASS_AVAILABLE(10_11, 9_0)
@interface CNContactFormatter : NSFormatter
```

/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Versions/C/Headers/NSDateFormatter.h
```
@interface NSFormatter : NSObject <NSCopying, NSCoding>
```

So, @spouliot  did we mess up the bindings? Or is there another place it can be added in obj-c beyond the @interface delcs? Do I need more ☕️ this morning already?

Reply:
```
NS_CLASS_AVAILABLE(10_11, 9_0)
@interface CNContactFormatter : NSFormatter <NSSecureCoding>
```

^ iOS SDK